usaspending_api/etl/subaward_etl.py

```diff
@@ -180,9 +180,16 @@ def load_subawards(submission_attributes, db_cursor):
         recipient = LegalEntity.objects.filter(recipient_unique_id=row['duns']).first()
 
         if not recipient and row['duns']:
+            recipient_name = row['awardee_name']
+            if recipient_name is None:
+                recipient_name = row['awardee_or_recipient_legal']
+            if recipient_name is None:
+                logger.warn("Subaward number {} has no recipient name; skipping...".format(row['subaward_num']))
```

Comment:
I think if both name fields are null we should probably skip, unless there's a DUNS. Otherwise how useful is the record?